app/navigation/Navigator.scala

```diff
@@ -61,8 +61,6 @@ class Navigator @Inject() () {
     case ProofOfAuthorityPage                       => _ => routes.BankDetailsController.onPageLoad(NormalMode)
     case CheckYourAnswersPage                       => _ => routes.ConfirmationController.onPageLoad()
     case AmendCheckYourAnswersPage                  => _ => routes.AmendConfirmationController.onPageLoad()
```

Comment:
CLARIFICATION: Is AmendCheckYourAnswersPage = AmendConfirmationController supposed to still be in the Navigator class?

Reply:
No.    I only deleted what was not compiling and not needed.  There's probably more not needed stuff in there.  Eventually this whole class will be removed of course, but I'll have another "sweep".

Reply:
Removed unused code from old navigator